general/g.tempfile/main.c

```diff
@@ -52,6 +53,13 @@ int main(int argc, char *argv[])
     dry_run->description =
         _("Dry run - don't create a file, just prints it's file name");
 
+    directory = G_define_flag();
+    directory->key = 'f';
```

Comment:
d is unfortunately used, but f is not great: folder, file, or directory?

Reply:
I felt the same but had no better idea. I considered `-D` but that did not seem better than `-f` for folder mode...
Any suggestions?